masterdata_context.md

```diff
@@ -2,36 +2,36 @@
+Type | GREM Status | REM Status | Description
+---|---|---|---
+[Masterdata.Property.Created](#masterdatapropertycreated) | | Draft 9.2 | A new property has been created
+[Masterdata.Property.Updated](#masterdatapropertyupdated) | | Draft 9.2 | Data associated to a property has changed; you get changed attributes only
+[Masterdata.Property.TagAdded](#masterdatapropertytagadded) | | N/A | A tag was added to a property; please read the specs for this event carefully
+[Masterdata.Property.TagRemoved](#masterdatapropertytagremoved) | | N/A | A tag was removed from a property; please read the specs for this event carefully
+[Masterdata.Building.Created](#masterdatabuildingcreated) | | Draft 9.2 | A building has been created
+[Masterdata.Building.Updated](#masterdatabuildingupdated) | | Draft 9.2 | Data associated to a building has changed; you get the reference plus all changed attributes
+[Masterdata.Building.Deleted](#masterdatabuildingdeleted) | | Draft 9.2 | The building was deleted
+[Masterdata.Unit.Created](#masterdataunitcreated) | | Draft 9.2 | A rentable unit has been created
+[Masterdata.Unit.Updated](#masterdataunitupdated) | | Draft 9.2 | Data associated to a rentable unit has changed; you get the reference plus all changed attributes
+[Masterdata.Unit.Deleted](#masterdataunitdeleted) | | Draft 9.2 | The unit was deleted
+[Masterdata.Unit.ReferenceChanged](#masterdataunitreferencechanged) | | Draft 9.2 | The unit reference has changed
+[Masterdata.ManagementTeam.Updated](#masterdatamanagementteamupdated) | | N/A | A change to a property management team was applied; only changed roles are published
+[Masterdata.Configuration.SedexIdChanged](#masterdataconfigurationsedexidchanged) | Draft | 9.1 | A new SedexID has been configured |
 
 ### Masterdata.Property.Created
 
-Field | Type | Content / Remarks
----|---|---
-eventType | string | Masterdata.Property.Created
-data | hash |
-&nbsp;&nbsp;reference | string | unique identifier for the property
-&nbsp;&nbsp;description | string |
-&nbsp;&nbsp;zipCode | string |
-&nbsp;&nbsp;city | string |
-&nbsp;&nbsp;countryCode | string | ISO country code, eg 'CH'
-&nbsp;&nbsp;mandateTerminatedBy | string | ISO 8601 encoded date, eg '2019-03-01'
-&nbsp;&nbsp;startOfAdministration | string | ISO 8601 encoded date, eg '2019-03-01'
-&nbsp;&nbsp;endOfAdministration | string | ISO 8601 encoded date, eg '2019-03-01'
+Field | Type | GREM Status | REM Status | Content / Remarks
+---|---|---|---|---
+eventType | string | | Draft 9.2 | Masterdata.Property.Created
+data | hash | | |
+&nbsp;&nbsp;reference | string | | Draft 9.2 | unique identifier for the property
+&nbsp;&nbsp;description | string | | Draft 9.2 |
+&nbsp;&nbsp;zipCode | string | | Draft 9.2 |
+&nbsp;&nbsp;city | string | | Draft 9.2 |
+&nbsp;&nbsp;countryCode | string | | Draft 9.2 | ISO country code, eg 'CH'
+&nbsp;&nbsp;mandateTerminatedBy | string | | N/A | ISO 8601 encoded date, eg '2019-03-01'
```

Comment:
Should get a deprecated note (in all messages)

Reply:
For all 4 properties?

Reply:
No, only mandateTerminatedBy should be marked as deprecated since it is replaced by startOfAdministration but we cannot simply remove it, it is used in production.

Reply:
Ok.